ci/test.yml

```diff
@@ -225,6 +225,7 @@ transport manual:
       - TEST_FILE: ["methods"]
       - TEST_FILE: ["popup-close"]
       - TEST_FILE: ["popup-error-page"]
+      - TEST_FILE: ["third-party-close"]
```

Comment:
The reason I created new test is that in an `afterEach` there is a test flow that it is not possible when `third-party-close` because there is no third party anymore. But I guess it could be just added as different test after without that `afterEach`.

Reply:
I will try that, since you are right that we could save some time.

Reply:
So, I added this test to `packages/connect-popup/e2e/tests/popup-close.test.ts` skipping the afterEach for this one.
And it works.